### ISSUE TITLE
Extra JS in wp_head

### ISSUE THREAD

**Issue by GoogleCodeExporter** [viktorthang/wordpress-custom-content-type-manager#342]
```
Is there a way to remove the extra JS included by CCTM in the wp_head
```

Original issue reported on code.google.com by `kb6...@gmail.com` on 29 Feb 2012 at 2:40
- Merged into: #331

**Comment by GoogleCodeExporter** [comment]
```
<script type='text/javascript'>
/* <![CDATA[ */
var thickboxL10n = {"next":"Next >","prev":"< 
Prev","image":"Image","of":"of","close":"Close","noiframes":"This feature 
requires inline frames. You have iframes disabled or your browser does not 
support 
them.","loadingAnimation":"\/wp-includes\/js\/thickbox\/loadingAnimation.gif","c
loseImage":"/wp-includes\/js\/thickbox\/tb-close.png"};
/* ]]> */
</script>
<script type='text/javascript' 
src='/wp-includes/js/thickbox/thickbox.js?ver=3.1-20111117'></script>
<script type='text/javascript' 
src='/wp-admin/js/media-upload.js?ver=20110930'></script>
<script type='text/javascript' 
src='/wp-content/plugins/custom-content-type-manager/js/summarizeposts.js?ver=3.
3.1'></script>
<script type='text/javascript' 
src='/wp-content/plugins/custom-content-type-manager/js/post_widget.js?ver=3.3.1
'></script>


```

Original comment by `kb6...@gmail.com` on 29 Feb 2012 at 2:42

**Comment by GoogleCodeExporter** [comment]
```
Please always include your information when reporting a bug.  Help me help you. 
 In any CCTM admin page, there's a link for filing a bug that includes all your 
system info -- I put that in there because it  wastes a lot of my time going 
back and forth with vague nondescript reports of bugs.  Thanks.

Also, please provide details and steps to reproduce, otherwise there's no way 
for me to know what you're talking about.  If you take the time to file a bug, 
please take the time to do it correctly.  I'm guessing that this is a duplicate 
of issue 331, which has already been  corrected on the dev branch and will be 
released in 0.9.5.11.  If not, please re-post steps to reproduce.  Thanks.
```

Original comment by `ever...@fireproofsocks.com` on 29 Feb 2012 at 3:38
- Changed state: **Duplicate**
